app/models/user.rb

```diff
@@ -10,6 +10,8 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable,
          :invitable, :confirmable
 
+  has_many :uploads, dependent: :destroy
```

Comment:
```suggestion
  has_many :uploads
```

I don't think we want to remove the upload if the user is deleted (for whatever reason... they leave the org or something) ((although I think rubocop will have something to say about this...))

Reply:
I've added this in a commit.